README.md

```diff
@@ -53,6 +53,25 @@ Add it to your configuration:
 echo "[secret key]" > config/credentials/development.key
 ```
 
+### Add efile resources locally
+
+In development, we need to download the IRS e-file schemas zip manually from S3.
+
+> ℹ️ We avoid storing them in the repo because the IRS asked us nicely to try to limit distribution.
+
+- If you have AWS credentials (ask a fellow developer on the team) download the file `vita-min-irs-e-file-schema-prod/efile1040x_2020v5.1.zip`
```

Comment:
You don't have to download from s3 anymore! the script now suggests downloading it from google docs.

Reply:
Awesome! We updated the docs to reference google drive links instead of AWS.